continuous_integration/environment-3.10-dev.yaml

```diff
@@ -4,7 +4,8 @@ channels:
 - nodefaults
 dependencies:
 - dask>=2022.3.0
-- fastapi>=0.69.0
+# FIXME: handling is needed for httpx-based fastapi>=0.87.0
+- fastapi>=0.69.0,<0.87.0
```

Comment:
`fastapi>=0.87.0` pulls in `starlette>=0.21.0`, which switched to using `httpx` instead of `requests` for its `TestClient` API (https://github.com/encode/starlette/pull/1376); it's not immediately obvious to me how to support both versions of the API, so this pins us to the older `starlette` packages for now.